code/controllers/configuration.dm

```diff
@@ -455,11 +454,12 @@
 				if("cross_comms_name")
 					cross_name = value
 				if("panic_server_name")
-					panic_server_name = value
+					if (value != "\[Put the name here\]")

```

Comment:
That red mark is the github lexer.

technically, `\]` doesn't exist as a supported escape sequence because its not needed, you only technically need to escape the open, it ignores mismatched closing brackets. byond unofficially supports `\]` for readability considerations